ray-operator/config/samples/ray_v1alpha1_rayservice-alb-ingress.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ray-cluster-ingress
+  annotations:
+    alb.ingress.kubernetes.io/scheme: internet-facing
+    alb.ingress.kubernetes.io/tags: Environment=dev,Team=test
+    # See `ingress.md` for more details about how to choose subnets.
+    alb.ingress.kubernetes.io/subnets: subnet-d9d4ff92, subnet-0c0209db843c9fd72
+    alb.ingress.kubernetes.io/target-type: ip
+    # Health Check Settings
+    alb.ingress.kubernetes.io/healthcheck-protocol: HTTP
+    alb.ingress.kubernetes.io/healthcheck-port: traffic-port
```

Comment:
I do not have experience with configuring health checks for ALB. Based on [ALB's document](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.5/guide/ingress/annotations/), it said that "When using `target-type: instance` with a service of type NodePort, the healthcheck port can be set to `traffic-port` to automatically point to the correct port.". However, the default serve service type is ClusterIP rather than NodePort. In addition, the `target-type` is `ip` here rather than `instance`. Do I misunderstand anything? Have you ever deployed the YAML file on EKS?